tests/test_decoding.py

```diff
@@ -262,7 +285,18 @@ def test_decode_request_coded_const(self) -> None:
             long_name=None,
             description=None,
             admin_data=None,
+            protocol_snrefs=[],
+            related_diag_comm_refs_raw=[],
+            diagnostic_class=None,
+            is_mandatory_raw=None,
+            is_executable_raw=None,
+            is_final_raw=None,
             semantic=None,
+            comparam_refs=NamedItemList[CommunicationParameterRef](),
+            is_cyclic_raw=None,
+            is_multiple_raw=None,
+            addressing_raw=None,
+            transmission_mode_raw=None,
```

Comment:
the number of arguments that needs to be passed is getting ridiculous

Reply:
true. maybe we should instantiate these classes using their `.from_et()` constructors...